etl.js

```diff
@@ -2,43 +2,36 @@
 
 
 var lineReader = require('readline').createInterface({
-    input: require('fs').createReadStream('input.json')
+    input: require('fs').createReadStream('input')
 });
 
-var cat = {
-    "baby & child": 0,
-    "beauty": 1,
-    "diet & nutrition": 2,
-    "health & medicine": 3,
-    "home health care": 4,
-    "household & grocery": 5,
-    "personal care": 6,
-    "sexual health": 7,
-    "vitamins": 8
-};
-
 lineReader.on('line', function (line) {
     var value = JSON.parse(line);
     if (value["_type"] === "search") {
         var search = value["_source"].search;
         if (search.totalRecordCount > 0 && search.origin.search) {
             var query = search.query;
-            var tokens = query.split(" ");
-            if (query && tokens.length == 1) {
+            // var tokens = query.split(" ");
+            if (query) {
                 var availableNavigation = search.availableNavigation;
                 var nav1 = availableNavigation[0];
                 if (nav1.name === "categories.1") {
                     var category = [];
+                    var totalCount = 0;
                     for (var i = 0; i < nav1.refinements.length; i++) {
+                        var refinement = nav1.refinements[i];
+                        totalCount += refinement.count;
+                    }
 
+                    for (var i = 0; i < nav1.refinements.length; i++) {
                         var refinement = nav1.refinements[i];
                         category.push({
-                            "value": refinement,
-                            "percentage": refinement.count / search.totalRecordCount
+                            "value": refinement.value,
+                            "percentage": refinement.count / totalCount
                         });
                     }
                 }
-                var output = {"query": tokens[0], "category": category};
+                var output = {"query": query, "category": category};
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/groupby/Tax-O-nyms/pullRequest?prid=7877234)
```suggestion
                var output = {query, "category": category};
```